### PR TITLE
Updated property controls readme

### PR DESCRIPTION
**Pull request by kbondanza**
Updated property controls readme to use getInitialState, and types from `@matthamlin/property-controls`

### CODE REVIEW

File: packages/property-controls/README.md [hunk 1]
```diff
@@ -33,28 +31,25 @@ function Avatar({
...
     label: `The dimensions of the avatar component`,
     default: 40
   }
 }
 
-const {PropertyControls, initialState} = createControls({
-  inputs: UI,
-  propertyControls: Avatar.propertyControls,
-})
+const PropertyControls = getInitialState(Avatar.propertyControls)
```
Comment (kbondanza) [inline packages/property-controls/README.md]: hi @hamlim,

I'm a little confused with to use the getInitialState method here with the example of:

```
function App() {
let [state, dispatch] = useReducer(initialState, reducer);
return (
  <PropertyControls
      state={state}
      dispatch={dispatch}
    />
  )
}
render(<App />)
```

and then the PropertyControls function created below on line 82.

Thanks!

File: packages/property-controls/README.md [hunk 2]
```diff
@@ -33,30 +31,26 @@ function Avatar({
...
   }
 }
 
-const {PropertyControls, initialState} = createControls({
-  inputs: UI,
-  propertyControls: Avatar.propertyControls,
-})
-
 function App() {
+  const initialState = getInitialState(Avatar.propertyControls);
```
Comment (hamlim) [inline packages/property-controls/README.md]: Nit: We can define this outside of the App component so we only need to call it once

File: packages/property-controls/README.md (hunk 1 above)
Comment (hamlim) [reply packages/property-controls/README.md]: Did you resolve this already? Not sure I understand the question

File: packages/property-controls/README.md (hunk 2 above)
Comment (kbondanza) [reply packages/property-controls/README.md]: thanks! updated

File: packages/property-controls/README.md (hunk 1 above)
Comment (kbondanza) [reply packages/property-controls/README.md]: Maybe, sorry it's been a bit of a learning curve for me. Refactoring my app a bit, and will let you know. Thanks!

**Pull request merged**